diff --git a/bindings/Makefile.am b/bindings/Makefile.am
index 8dd02f0..e40f06c 100644
--- a/bindings/Makefile.am
+++ b/bindings/Makefile.am
@@ -42,7 +42,8 @@ Hamlib-pl.mk: $(srcdir)/Makefile.PL
 			CC="$(CC)"  \
 			OBJECT="hamlibperl_wrap.o" \
 			VERSION="$(PACKAGE_VERSION)" \
-			LIBS="-L$(top_builddir)/src/.libs -lhamlib"
+			LIBS="-L$(top_builddir)/src/.libs -lhamlib" \
+			INSTALLDIRS=vendor
 	sed -e 's/^LD_RUN_PATH.*/LD_RUN_PATH =/' Hamlib-pl.mk > Hamlib-pl-norpath.mk
 	mv Hamlib-pl-norpath.mk Hamlib-pl.mk
 
@@ -63,7 +64,7 @@ distcheck-perl: Hamlib-pl.mk
 	$(MAKE) $(AM_MAKEFLAGS) -f Hamlib-pl.mk distcheck
 
 install-perl: Hamlib-pl.mk
-	$(MAKE) $(AM_MAKEFLAGS) -f Hamlib-pl.mk install_site
+	$(MAKE) $(AM_MAKEFLAGS) -f Hamlib-pl.mk pure_install
 
 ## ExtUtils::MakeMaker says uninstall is deprecated and won't remove the files
 ## anyway, so we'll have to figure out another way to remove the installed files.
